Bottom: 43088d7aeaad9f99f0fd915dce10fa1af80caa08
Top:    a4660000ac48ec7a130da623e35f929e45d759e3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-11 16:30:50 -0500

Refresh of buildbot-add-buildbot-master

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 9781ada..45477f9 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -11,10 +11,25 @@ container_addr_map:
 container_port_map:
   buildbot:
     port_args: 
-      - '{{ container_addr_map.buildbot.ip_addr }}:25:25'
-      - '{{ container_addr_map.buildbot.ip_addr }}:465:465'
-      - '{{ container_addr_map.buildbot.ip_addr }}:587:587'
+      - '{{ container_addr_map.buildbot.ip_addr }}:443:443'
 
 # Network names
-docker_network_backend: 'backend'
 docker_network_frontend: 'frontend'
+
+buildbot_data_path: '/buildbot-data'
+buildbot_database: 'sqlite:///{{ buildbot_data_path }}/state.sqlite'
+buildbot_timezone: 'America/Chicago'
+
+# the name of the image being duilt and used for the container
+buildbot_image_repo: https://github.com/novatechweb/docker-buildbot.git
+buildbot_image_name: '{{ docker_registry_username }}/buildbot'
+
+# the name of the container being started
+buildbot_container_name: '{{ docker_name_prefix }}buildbot'
+
+# the name of the data-volume used by the container
+buildbot_dv_name: '{{ docker_name_prefix }}buildbot_DV'
+
+# restore directories to temporarly store data being restored into docker containers
+buildbot_docker_backup_dir: '{{ docker_backup_dir }}/buildbot'
+host_buildbot_docker_restore_dir: '{{ bacula_dest }}{{ buildbot_docker_backup_dir }}'
diff --git a/roles/buildbot-master/env.yml b/roles/buildbot-master/env.yml
new file mode 100644
index 0000000..7ef0578
--- /dev/null
+++ b/roles/buildbot-master/env.yml
@@ -0,0 +1,22 @@
+BUILDBOT_VERSION=1.1.1
+BUILDBOT_UID=1000
+
+BUILDBOT_VOL_CACHE=/home/coopera/Developer/buildbot-mounts/cache:/cache
+BUILDBOT_VOL_MASTER=/home/coopera/Developer/buildbot-mounts/buildbot:/buildbot
+BUILDBOT_VOL_NTEL=/home/coopera/Developer/buildbot-mounts/worker-ntel:/buildbot
+BUILDBOT_VOL_PTXDIST=/home/coopera/Developer/buildbot-mounts/worker-ptxdist:/buildbot
+BUILDBOT_VOL_SECRETS=./secrets:/run/secrets:ro
+
+BUILDBOT_DATABASE=sqlite:////cache/state.sqlite
+BUILDBOT_MASTER=buildbot
+BUILDBOT_TIMEZONE=America/Chicago
+BUILDBOT_HOST_PORT=172.16.103.71:80
+BUILDBOT_WEB_PORT=8010
+BUILDBOT_WEB_URL=http://buildbot.novatech-llc.com/
+
+BUILDBOT_WORKER_NTEL=worker-ntel
+BUILDBOT_WORKER_PASSWORD=pass
+BUILDBOT_WORKER_PORT=9989
+BUILDBOT_WORKER_PTXDIST=worker-ptxdist
+
+BUILDBOT_ENVIRONMENT_BLACKLIST=DOCKER_BUILDBOT*,BUILDBOT_ENV_*,BUILDBOT_1*,BUILDBOT_ENVIRONMENT_BLACKLIST,BUILDBOT_WORKER_PASSWORD
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 58c0cfa..5cf3e23 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -10,9 +10,21 @@
   docker_volume:
     name: '{{ buildbot_dv_name }}'
 
+- name: Checkout image repo
+  git:
+    repo: '{{ exim4_image_repo }}'
+    version: master
+    dest: '{{ docker_projects_dir }}/docker-exim4'
+
+- name: Create buildbot master image
+  docker_image:
+    name: '{{ buildbot_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/buildbot-master'
+
 - name: Start buildbot container
   docker_container:
     name: '{{ buildbot_container_name }}'
     image: '{{ buildbot_image_name }}:{{ docker_image_tag }}'
     volumes:
-      - '{{ buildbot_dv_name }}:/buildbot-data:z'
+      - '{{ buildbot_dv_name }}:{{ buildbot_data_path }}:z'
diff --git a/roles/buildbot-master/vars/main.yml b/roles/buildbot-master/vars/main.yml
index 62beac8..eed3963 100644
--- a/roles/buildbot-master/vars/main.yml
+++ b/roles/buildbot-master/vars/main.yml
@@ -1,16 +1,2 @@
 ---
 # vars file for buildbot
-
-# the name of the image being duilt and used for the container
-buildbot_image_repo: https://github.com/novatechweb/docker-buildbot.git
-buildbot_image_name: '{{ docker_registry_username }}/buildbot'
-
-# the name of the container being started
-buildbot_container_name: '{{ docker_name_prefix }}buildbot'
-
-# the name of the data-volume used by the container
-buildbot_dv_name: '{{ docker_name_prefix }}buildbot_DV'
-
-# restore directories to temporarly store data being restored into docker containers
-buildbot_docker_backup_dir: '{{ docker_backup_dir }}/buildbot'
-host_buildbot_docker_restore_dir: '{{ bacula_dest }}{{ buildbot_docker_backup_dir }}'
